src/main/kotlin/io/github/yuutoproject/yuutobot/utils/okHttpClientInstance.kt

```diff
@@ -0,0 +1,23 @@
+/*
+ * Open source bot built by and for the Camp Buddy Discord Fan Server.
+ *     Copyright (C) 2020  Kyuuto-devs
+ *
+ *     This program is free software: you can redistribute it and/or modify
+ *     it under the terms of the GNU General Public License as published by
+ *     the Free Software Foundation, either version 3 of the License, or
+ *     (at your option) any later version.
+ *
+ *     This program is distributed in the hope that it will be useful,
+ *     but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *     GNU General Public License for more details.
+ *
+ *     You should have received a copy of the GNU General Public License
+ *     along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.github.yuutoproject.yuutobot.utils
+
+import okhttp3.OkHttpClient
+
+val httpClient = OkHttpClient()
```

Comment:
I am not a fan of this, what does it compile to?

Reply:
In your ide `tools > kotlin > show kotlin bytecode` and then press the decompile button in that window

Reply:
`public final class OkHttpClientInstanceKt {
   @NotNull
   private static final OkHttpClient httpClient = new OkHttpClient();

   @NotNull
   public static final OkHttpClient getHttpClient() {
      return httpClient;
   }
}
`

Reply:
I don't know the proper way to implement it on kotlin, can you show me?

Reply:
I'm happy with what it compiles to, it is what I would want it to compile to